src/TypeScript/Analyzer/EslintBridgeIssueConverter.cs

```diff
@@ -32,28 +33,29 @@ internal interface IEslintBridgeIssueConverter
     }
 
     internal delegate string ConvertToSonarRuleKey(string eslineRuleKey);
+    internal delegate IEnumerable<RuleDefinition> GetRuleDefinitions();
 
     internal class EslintBridgeIssueConverter : IEslintBridgeIssueConverter
     {
         private readonly ConvertToSonarRuleKey keyMapper;
+        private readonly GetRuleDefinitions ruleDefinitionProvider;
 
-        public EslintBridgeIssueConverter(ConvertToSonarRuleKey keyMapper)
+        public EslintBridgeIssueConverter(ConvertToSonarRuleKey keyMapper, GetRuleDefinitions ruleDefinitionProvider)
```

Comment:
Strictly speaking this should be using the current active rule configuration (assuming we allow the user to change the severity; they can for C#, and I think they can for C++).
However, we can change that when we support user configuration.